AuthoringTests.md

```diff
@@ -49,11 +113,11 @@ as the base class, because NUnit treats namespaces like test suites, and we have
 
 3. `TestsWithEnlistmentPerTestCase`
 
-   Derive from this class if you need a brand new enlistment per test case.  This is the most reliable, but also most expensive option.
+   Derive from this class if you need a brand-new enlistment per test case.  This is the most reliable, but also most expensive option.

```

Comment:
I would consider removing `brand` altogether and have it say "you need a new enlistment created for each test case."